enhancements/proposals/NNNN-alternator/README.md

```diff
@@ -0,0 +1,164 @@
+# Support for ScyllaDB Alternator
+
+## Summary
+
+This proposal aims to introduce proper support for enabling Alternator for ScyllaClusters 
+and deal with the legacy API fields used to partially enable it only in an insecure manner
+which also prevents us from serving on any new port within the ScyllaDB pod in the future 
+without breaking existing deployments.
+
+## Motivation
+
+Alternator is an important service provided by ScyllaDB for DynamoDB compatible API
+that we need to support in the operator.
+We have historical and limited option to enable it but only for unencrypted traffic.
+There are no e2e tests for the existing functionality.
+Also, the user specified (variable) port means that when we need to add a new port to the sidecar,
+we can break existing deployments. Same goes for the Service port.
+
+### Goals
+
+- Expose Alternator service using a load balanced endpoint (Service)
+- Expose Alternator over secure and static port
+- Expose Alternator on insecure port only if explicitly requested
+- Ensure e2e coverage for Alternator
+- Expose Alternator together with CQL (not exclusively)
+- Deprecate `scyllaclusters.scylla.scylladb.com/v1.spec.alternator.port`
+- Enable transition period and be backwards compatible with existing clusters
+
+### Non-Goals
+
+- Exposing Alternator on every node individually and relying on client side load balancing
+- Supporting host networking tweaks
+- Disabling CQL when Alternator is enabled
+- Enabling Alternator on insecure port only
+
+## Proposal
+
+I propose to expose Alternator using the existing identity Service (`<sc-name>-client`) on port `8001` (https).
```

Comment:
Current logic adds additional ports to every member Services when Alternator is enabled. Proposal doesn't cover whether it's going to be changed or not.

Reply:
Added explicit mention of retaining the existing behaviour as implied by backwards compatibility.